GoogleUtilities/AppDelegateSwizzler/GULAppDelegateSwizzler.m

```diff
@@ -239,17 +239,13 @@ + (void)unregisterAppDelegateInterceptorWithID:(GULAppDelegateInterceptorID)inte
 
 + (void)proxyOriginalDelegate {
   dispatch_once(&sProxyAppDelegateOnceToken, ^{
-    id<UIApplicationDelegate> originalDelegate = [self sharedApplication].delegate;
+    id<UIApplicationDelegate> originalDelegate = [GULAppDelegateSwizzler sharedApplication].delegate;
     [GULAppDelegateSwizzler proxyAppDelegate:originalDelegate];
   });
 }
 
 #pragma mark - Create proxy
 
-/** Returns the current sharedApplication.
- *
- *  @return the current UIApplication if in an app, or nil if in extension or if it doesn't exist.
- */
 + (UIApplication *)sharedApplication {
   // YES if the bundle is an app extension.
   if ([[NSBundle mainBundle].bundlePath hasSuffix:@".appex"]) {
```

Comment:
This is not depending on Core - we will need to add a dependency on the Environment subspec of GoogleUtilities to be able to use this!

Reply:
I guess that's fine probably not worth well to depend on core just for this.

Reply:
Should be fine for Swizzler to depend on GoogleUtilities/Environment since it always gets pulled into any Firebase app anyway.

Doing so will add consistency and a bit of code size reduction.

Reply:
Done.